UpdateCgnOrchestrator/index.ts

```diff
@@ -86,24 +83,24 @@ export const handler = function*(
     trackExAndThrow(e, "cgn.update.exception.decode.input")
   );
 
-  const { fiscalCode, newStatus } = decodedInput;
+  const { fiscalCode, newStatusCard } = decodedInput;
```

Comment:
We send newStatus as Orchestrator input, see [here](https://github.com/pagopa/io-functions-cgn/pull/19/files#diff-67d3faabf9f0f8527de98c0d5f96ccc231b8d912b3633ed6a154328137af1e88R87)
```suggestion
  const { fiscalCode, newStatus } = decodedInput;
```